api/leaderboard.go

```diff
@@ -557,10 +559,17 @@ func (app *App) GetTopMembers(ctx context.Context, req *api.GetTopMembersRequest
 
 	tenantID := metadata.ValueFromIncomingContext(ctx, "tenant-id")
 	if tenantID != nil {
-		members, err = app.Enricher.Enrich(tenantID[0], req.LeaderboardId, members)
+		result, err := app.Enricher.Enrich(ctx, tenantID[0], req.LeaderboardId, members)
 		if err != nil {
-			lg.Error("Enriching members failed.", zap.Error(err))
-			return nil, status.Errorf(codes.Internal, "Unable to enrich members")
+			if errors.Is(err, enriching.ErrNotConfigured) {
```

Comment:
We could also invert the logic here to avoid the if/else, WDYT?

 ```suggestion
			if !errors.Is(err, enriching.ErrNotConfigured) {
				lg.Error("Enriching members failed.", zap.Error(err))
				return nil, status.Errorf(codes.Internal, "Unable to enrich members")
                         }
                         
                         lg.Debug("Enrichment not configured.", zap.Error(err))
		}
```

Reply:
ignore the broken suggested code haha

Reply:
Done